src/components/BrowseResultsPane/BrowseResultsPane.js

```diff
@@ -38,7 +39,7 @@ const BrowseResultsPane = ({
 
   const isEmptyMessage = useMemo(() => (
     <NoResultsMessage
-      filters={filters}
+      filters={omit(filters || {}, ['qindex', 'query'])}
```

Comment:
If I'm not mistaken, lodash `omit` will return an empty object by default if the first parameter is not defined.

Reply:
good catch